FrostyModManager/App.xaml.cs

```diff
@@ -129,18 +129,26 @@ private void Application_Startup(object sender, StartupEventArgs e)
                     {
                         defaultConfig = new FrostyConfiguration(prof);
                     }
-                    catch (System.IO.FileNotFoundException)
+                    catch (FileNotFoundException)
                     {
+                        defaultConfig = null;
                         Config.RemoveGame(prof); // couldn't find the exe, so remove it from the profile list
+                        Config.Remove("DefaultProfile");
+                        Config.Save();
+                    }
+                    catch
+                    {
+                        defaultConfig = null;
+                        Config.Remove("DefaultProfile");
                         Config.Save();
                     }
                 }
-                else
-                {
-                    Config.Add("UseDefaultProfile", false);
-                    Config.Save();
-                }
-            }
+//                else
```

Comment:
does this cause any issues or why is it commented out?

Reply:
After an error occurs, user may not want to disable it. If thats wrong u can uncomment it

Reply:
no it makes sense, just remove it then